src/styles/base.scss

```diff
@@ -1306,3 +1306,638 @@ td, th {
   display: block;
   color: #4d4d4d;
 }
+
```

Comment:
I am not sure if all these styles are (or are going to be) used. In future PR's, @sanketbansal @Sanji515 try to remove redundant and unused styles from here.

Reply:
yeah, I think that we can do in other dependent PRs

Reply:
@lunayach yeah of course some changes are new but most of them are already on the master, this is the reason I'm saying to close this PR and do the css changes in their respective PR so that it would be easy to track what should not be included in the PR. These fonts can also be included in their respective PR's

Reply:
I don't think that the `base.scss` is having changes for the already pushed PR's. That should not be the case anyways. Is it correct, @sanketbansal?

Reply:
so, @Sanji515 can you tell me the changes which have already been made in the master?
Then I will remove them

Reply:
@lunayach I have tried to added changes related to my PRs.
Also, U and @Sanji515  can run the PR on local machine and check if its hindering with already made changes.

Reply:
Ah, let's not go this way then. Use updated master's `base.scss` for all your future PR's. 
Let's also remove `base.scss` from this PR. (That is, only static files for now.)
That way, I guess it would be easy to keep a track of all the changes done for a particular functionality.

Reply:
I have checked there is no duplicate styles present in base.scss file. Also no style is hindering with the master branch. Also these added styles are required by the other components.

Reply:
(IMO) if `base.scss` is also having styles for already pushed PR's, then we should not include it in this PR. We can easily afford one extra file change in the form of `base.scss` for future PR's.

Also, can you confirm for @Sanji515's comment:

> @sanketbansal The organizations' images that you put inside the organizations' folder are already there inside images/partners folder. So I think you should check for the duplicate images there and at other places also.

So, that means only static file changes for this PR. @sanketbansal I understand that you had to make lot's of changes for this (and other related) PR's, but to keep the project structure clean, I guess, it's worth it.

Reply:
That's why I was keeping a separate file of my changes. And was looking to add that into base.scss after all PRs got merged.

Reply:
But can we add those changes PR by PR as we proceed (in `base.scss`)?

Reply:
Any issue with variable.scss file?
Also, should I make changes in base.scss in other PRs or not?

Reply:
`Variable.scss` is fine.
Sure, you can make changes in `base.scss` for the functionality that the other PR introduces.

Reply:
> But can we add those changes PR by PR as we proceed (in `base.scss`)?

For that I have to check each and every style of the HTML in other PR and then have to search whether that is already present or not which is going to be a tedious task.

Reply:
I think to avoid possible duplicities and adding those things only which we actually need, it should be done. We could also take @RishabhJain2018's take on this.

Reply:
I agree with @lunayach.